docs/content/en/snippets/week4.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Week 4 Snippet"
+date: 2022-02-04T15:03:10-08:00
+draft: true
+# The following are extra (optional) metadata for ananke
+# tags: ["tag1", "tag2"]
+featured_image: "https://images.unsplash.com/photo-1531403009284-440f080d1e12?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2340&q=80"
+author: "Kyle Zhang"
+description: "Hello Midterm"
+---
+
+Team members work on their milestone towards the MVP next week.
+
+<!--more-->
+
+## New Features/Functionalities
+
+- Brushes with size adjustment
+- Eraser
+- 3D Object Translation
+- 3D Object Sclaing
+
+## Review Requests
+
+- Nothing for now.
```

Comment:
I think https://github.com/UWRealityLab/xrcapstone22wi-team8/pull/14 should be ready for review? Or are we waiting for it to be merged first?